### DISCUSSION TITLE
Compatibility with File Tree Alternative Plugin

### DISCUSSION THREAD

**Discussion by mangost23** [snezhig/obsidian-front-matter-title#205] [Ideas]
Hi, the plugin does not work with the File True Alternative Plugin. Please fix it.
https://github.com/ozntel/file-tree-alternative

**Comment by snezhig** [comment]
Hi, if you want support for this plugin, then you need to contact its author so that he can add support via api.
https://github.com/snezhig/front-matter-plugin-api-provider
